### PR TITLE
Add new section to update server xml

### PR DESCRIPTION
**Pull request by Kevcpro**

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
@Kevcpro 
> ...fix the hotspot that it does not highlight the line `<feature>beanValidation-2.0</feature>`

I mean that the `beanValidation` hotspot is needed. In your previous change, it does not work. I expect that you can fix the hotspot to make it work.

**Comment by gkwan-ibm** [comment]
@Kevcpro 
would you update the following?
- **Programmatically validating constraints**
  - [x] "Finally, create a service..." -> "Next, create a service..."
- **Enabling the Bean Validation feature**
  - [x] "Enable the Bean Validation..." -> "Finally,  add the Bean Validation..."

**Review by davidwyao — APPROVED** [review]

**Review by davidwyao — APPROVED** [review]

File: README.adoc [hunk 1]
```diff
@@ -284,7 +284,7 @@ supplied serial number matches the constraint, [hotspot=isValid file=2]`isValid(
 // ==================================================================================
 == Programmatically validating constraints
 
-Finally, create a service that can be used to programmatically validate the constraints
+Next, create a service that can be used to programmatically validate the constraints
```
Comment (dmuelle) [inline README.adoc]: suggested edit:

Next, create a service to programmatically validate the constraints

File: README.adoc [hunk 2]
```diff
@@ -330,6 +330,25 @@ A call is then made to the [hotspot=launchSpacecraft file=0]`launchSpacecraft()`
...
+
+Finally, add the Bean Validation feature in the application by updating the `server.xml` configuration file.
+
+[role="code_command", subs="quotes"]
+----
+#Replace the `server` configuration file.#
+`src/main/liberty/config/server.xml`
+----
+
+The [hotspot=beanValidation file=0]`beanValidation-2.0` feature definition enables the bean validation feature in the
```
Comment (dmuelle) [inline README.adoc]: suggested edits

You can now use [hotspot=beanValidation file=0]`beanValidation-2.0` feature  to validate that the supplied JavaBeans meet the defined constraints.

**Review by dmuelle — CHANGES_REQUESTED** [review]
a couple suggestions for concision/ clarity

**Review by dmuelle — APPROVED** [review]
LGTM

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**